include/RAJA/index/IndexSetUtils.hpp

```diff
@@ -90,11 +84,12 @@ RAJA_INLINE void getIndicesConditional(CONTAINER_T& con,
                                        CONDITIONAL conditional)
 {
   CONTAINER_T tcon;
-  forall<ExecPolicy<seq_segit, seq_exec> >(iset,
-    [&](typename CONTAINER_T::value_type idx) {
-      if (conditional(idx)) tcon.push_back(idx);
-    }
-  );
+  forall<ExecPolicy<seq_segit, seq_exec>>(
+      iset,
+      [&](typename CONTAINER_T::value_type idx)
```

Comment:
This is what I would expect in my comments above. It appears that it is formatting lambda expressions in multiple inconsistent ways.....grrrfff.